tox.ini

```diff
@@ -18,7 +18,8 @@ commands =
     # -l: show locals in tracebacks
     # --tb=short: short traceback print mode
     # --strict: marks not registered in configuration file raise errors
-    pytest --cov=py42 --cov-report xml --cov-report term -v -rsxX -l --tb=short --strict
+    # -m="not integration": exclude tests that are marked as integration
+    pytest --cov=py42 --cov-report xml --cov-report term -v -rsxX -l --tb=short --strict -m"not integration"
```

Comment:
Do you need a space between the `m` and the `"not integration"` ?

Reply:
Changed.